### DISCUSSION TITLE
Possible to run reloadium in docker container?

### DISCUSSION THREAD

**Discussion by mbaeuerle** [reloadware/reloadium#14] [Q&A] [answered]
I tried to run reloadium from within a K8s Docker container to leverage hot reload capabilites when developing python from within the container. (If you're wondering, using [Garden](https://garden.io/) to sync the code).
However I get the following error:

> It seems like your platform or Python version are not supported yet.

I just wanted to get confirmation that running reloadium in a container is currently not supported. Or maybe I just did something wrong?

Thanks!

**Comment by dkrystki** [comment, marked as answer]
It should work fine as long there's a wheel for the system your docker image is based on.
What docker image are you using?
I've tested it on the standard python 3.9 one.

**Reply by mbaeuerle** [reply]
Ah that's it, thanks. It's a Python 3.9 Alpine 3.x image and therefore doesn't have a wheel. 
Testing with a standard python 3.9 works without issues!
